doc/man3/OSSL_QUIC_client_method.pod

```diff
@@ -0,0 +1,55 @@
+=pod
+
+=head1 NAME
+
+OSSL_QUIC_client_method, OSSL_QUIC_client_thread_method,
+OSSL_QUIC_server_method
+- Provide SSL_METHOD objects for QUIC enabled functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/quic.h>
+
+ const SSL_METHOD *OSSL_QUIC_client_method(void);
+ const SSL_METHOD *OSSL_QUIC_client_thread_method(void);
+ const SSL_METHOD *OSSL_QUIC_server_method(void);
+
+=head1 DESCRIPTION
+
+The OSSL_QUIC_client_method(), OSSL_QUIC_client_thread_method(), and
+OSSL_QUIC_server_method() functions provide methods for the
+L<SSL_CTX_new_ex(3)> function to provide QUIC protocol support.
+
+The OSSL_QUIC_client_thread_method() uses threads to allow blocking
+mode of operation and avoid the need to return the control to the
+OpenSSL library for processing time based events.
+The OSSL_QUIC_client_method() does not use threads and depends on
```

Comment:
_and depends on nonblocking mode_ might be better as _and depends on a nonblocking mode of operation_